src/amqp_direct_connection.erl

```diff
@@ -38,7 +38,7 @@
                 connected_at
                }).
 
--define(INFO_KEYS, [type]).
+-define(INFO_KEYS, [type, internal_user]).
```

Comment:
I'm on the fence about this. Exposing internal records is not a good idea. At the same time, direct connections are primarily used by plugins. We need to discuss this internally.